From: Gokturk Yuksek <gokturk@binghamton.edu>
Subject: [PATCH] Backport missing cookie support

For <gsoap-2.8.24, successful compilation results in missing cookie
support due to undefined compilation flag. This is fixed in the later
versions by upstream.

Upstream-URL: https://sourceforge.net/p/gsoap2/code/85/tree//gsoap/Makefile.am?diff=51af4d3f5fcbc945b53d89a4:84
Gentoo-Bug-URL: https://bugs.gentoo.org/show_bug.cgi?id=340647

--- a/gsoap/Makefile.am
+++ b/gsoap/Makefile.am
@@ -42,9 +42,9 @@
 libgsoapck___a_SOURCES = stdsoap2_ck_cpp.cpp
 libgsoapck___a_CXXFLAGS = $(SOAPCPP2_DEBUG) $(SOAPCPP2_NONAMESPACES) -D$(platform) -DWITH_COOKIES
 libgsoapssl_a_SOURCES = stdsoap2_ssl.c
-libgsoapssl_a_CFLAGS = $(SOAPCPP2_DEBUG) $(SOAPCPP2_NONAMESPACES) -D$(platform) -DWITH_OPENSSL -DWITH_DOM -DWITH_GZIP
+libgsoapssl_a_CFLAGS = $(SOAPCPP2_DEBUG) $(SOAPCPP2_NONAMESPACES) -D$(platform) -DWITH_OPENSSL -DWITH_DOM -DWITH_GZIP -DWITH_COOKIES
 libgsoapssl___a_SOURCES = stdsoap2_ssl_cpp.cpp
-libgsoapssl___a_CXXFLAGS = $(SOAPCPP2_DEBUG) $(SOAPCPP2_NONAMESPACES) -D$(platform) -DWITH_OPENSSL -DWITH_DOM -DWITH_GZIP
+libgsoapssl___a_CXXFLAGS = $(SOAPCPP2_DEBUG) $(SOAPCPP2_NONAMESPACES) -D$(platform) -DWITH_OPENSSL -DWITH_DOM -DWITH_GZIP -DWITH_COOKIES
 
 BUILT_SOURCES = stdsoap2_cpp.cpp $(lib_LIBRARIES)
 
